pkg/pillar/containerd/containerd.go

```diff
@@ -7,25 +7,27 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cio"
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/mount"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/snapshots"
+	"github.com/eriknordmark/netlink"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/opencontainers/image-spec/identity"
```

Comment:
You and @kalyan-nidumolu need to talk because he is moving these github imports to a separate block and the end and you just moved them in the other direction. I don't care which approach we take (a single alphabetical vs, two blocks with standard packages in first) but we shouldn't apply different approaches with the risk for flip-flop.

Reply:
Good point @eriknordmark . Hey @kalyan-nidumolu -- do you have any style guidance here here? I'm fine either way.